app/1.0/blog/2017-02-06-news-pwa.md

```diff
@@ -0,0 +1,21 @@
+---
+title:  "News Progressive Web App"
+
+---
+
+We're excited to release a new progressive web app demo and template built with Polymer - [Polymer News](https://news.polymer-project.org/list/top_stories)!
+
+The Polymer News progressive web app is designed to showcase the fast performance and slick user experience possible using modern web platform features, and serve as a jumping-off point for building a more custom and feature-rich app.
+
+It is built with Polymer 1.x (to be upgraded to [Polymer 2.0](https://www.polymer-project.org/2.0/docs/about_20) after 2.0 is released), and patterned after a typical "news" site - with categories and article pages.  It showcases a number of different application patterns:
+
+* It is built using the [Polymer CLI](https://www.polymer-project.org/1.0/docs/tools/polymer-cli) in order to [generate a Service Worker](https://www.polymer-project.org/1.0/toolbox/service-worker) to help it work offline.
```

Comment:
This sentence reads a little funny to me. Maybe break it in two (* Built with the Polymer CLI, * Uses a service worker.) ... Or skip the CLI part and just mention the service worker part.